src/plugins/open/open.c

```diff
@@ -221,113 +218,112 @@ ipmi_openipmi_send_cmd(struct ipmi_intf * intf, struct ipmi_rq * req)
 			intf->target_addr,intf->target_channel, intf->my_addr);
 
 		if(intf->transit_addr != 0 && intf->transit_addr != intf->my_addr) { 
-		   uint8_t index = 0;
+			uint8_t index = 0;
       
```

Comment:
This line contains stray spaces, and there are more like it. Please clean them all up while you're at it.